src/cc_catalog_airflow/dags/provider_api_scripts/cleveland_museum_of_art.py

```diff
@@ -0,0 +1,213 @@
+    ):
+        logger.warning(f'Bad response_json:  {response_json}')
+        logger.warning(
+            'Retrying:\n_get_response_json(\n'
+            f'    {endpoint},\n'
+            f'    {query_params},\n'
+            f'    {request_headers}'
+            f'    retries={retries - 1}'
+            ')'
+        )
+        response_json = _get_response_json(
+            query_params,
+            endpoint=endpoint,
+            request_headers=request_headers,
+            retries=retries - 1
+        )
+
+    return response_json
+
+
+
+
+def handle_the_response(response):
+    for data in response:
+        foreign_url = data.get('url', None)
+        single_image = data.get('images')
+        image_url = None
+        images_till_now = 0
+        if single_image is not None:
+            if single_image.get('web'):
+                key = 'web'
+                image_url = single_image.get('web').get('url', None)
+            elif single_image.get('print'):
+                key = 'print'
+                image_url = single_image.get('print').get('url', None)
+            else:
+                key = 'full'
+                image_url = single_image.get('full').get('url', None)
+            
+            width = single_image[key]['width']
+            height = single_image[key]['height']
+            foreign_id = data.get('id', image_url)
+            license_status = data.get('share_license_status', None).lower()
+            license_share = license_status.lower()
+            title = data.get('title', '')
+
+            creator_info = data.get('creators', {})
+            creator_name = None
+            if creator_info:
+                creator_name = creator_info[0].get('description', '')
+            meta_data = create_meta_data(data)
+            images_till_now = image_store.add_item(
+                foreign_url,  # Foreign Landing URl
+                image_url,  # Image URL
+                None,  # Thumbnail_URL
+                None,  # License_URL
+                license_status,  # License
+                "1.0",  # License_Version
+                foreign_id,  # Foreign Identifier
+                width,  # Width of the image
+                height,  # height of the image
+                creator_name,  # Creator Name
+                None,  # Creator URl
+                title,  # Title of the Image
+                meta_data,  # Meta Data
+                None,  # Raw Tags
+                'f',  # Watermarked
+                None,  # Source
+            )
+    return images_till_now
+
+
+
+def create_meta_data(data):
+    meta_data = {}
+    meta_data['accession_number'] = data.get('accession_number', '')
+    meta_data['technique'] = data.get('technique', '')
+    meta_data['date'] = data.get('date', '')
+    meta_data['credit_line'] = data.get('creditline', '')
+    meta_data['classificatoin'] = data.get('type', '')
+    meta_data['culture'] = ','.join(list(filter(
+        None, data.get('culture', ''))))
```

Comment:
The fallback in the `get` here should be an empty list, not a string.  Also, please consider using a list comprehension instead:
```python
    meta_data['culture'] = ','.join(
        [i for i in data.get('culture', []) if i is not None]
    )
```